modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,6 +1250,50 @@ endif::[]
 
 == Disaster recovery
 
+With tiered storage enabled, you can restore your cluster, including its metadata, in the event of failure. This is a faster, less expensive alternative to active-active replication (for example, using xref:upgrade:migrate/data-migration.adoc[MirrorMaker 2]).
+
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster
+. Recover metadata from a failed source cluster to the new cluster
+. Recover topic data from the source cluster to the new cluster
+. Perform a rolling restart, if necessary
+. Direct application workload to the new cluster
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. Note that this method does not guarantee consistency at the application level. Restored transactions may not uphold atomicity. The consumer offset topic may not reflect point-in-time snapshots of topic data already consumed by applications. 
+
+=== Start a recovery target cluster
+
+Follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. Cloud account credentials (such as secrets or access keys) are not included in backed up metadata, and so you must manually configure these credentials for the new cluster.
+
+=== Cluster metadata recovery
+
+Metadata from a failed cluster can be restored to a target cluster using an rpk command. 
+
+[,bash]
+----
+rpk 
+----
+
+The following metadata is included in the restore:
+
+- Topic definitions. If you have enabled tiered storage only for specific topics, non-tiered storage will not be recovered 
+- Users and access control lists (ACLs)
+- Schemas
+- Consumer offsets 
+- All cluster configurations, including your Redpanda license key
+- Transaction metadata, up to the highest committed transaction. In-flight transactions are treated as aborted and will not be included in recovery
+
+NOTE: Committed transactions are not guaranteed to be atomic.
+
+The destination cluster must be empty; that is, the cluster cannot contain data other than node and cluster configuration properties. If metadata indicating usage is found in the destination (such as user-provided schemas, non-superusers, ACLs, consumer offsets, or user-created topics), the restore will fail and the rpk command will issue a warning.
+
+If duplicate metadata is found in the destination cluster, it will be overwritten by the restore.
```

Comment:
@andrwng to be clear here we only mean 'duplicate cluster configuration', right?

Reply:
Hmm, yeah I'm not quite sure what this is referring to. @kbatuigas could you clarify? Or maybe remove

Reply:
@mattschumpert @andrwng yes, I lifted it from the [PRD](https://docs.google.com/document/d/1840e92oD8CPifNqCMt7KN9Z2PH5Ej-mceSJN9WS3Olc/edit#heading=h.7i0ls36lt6ow) but happy to remove if appropriate

Reply:
@andrwng should we say 'duplicate cluster configuration' or not?  (duplicate config props will be overridden by restore, right)?

Reply:
Right, the cluster configuration is overridden by restore